src/scss/theme/_headings.scss

```diff
@@ -2,7 +2,7 @@
 
 // Headings
 h1, h2, h3, h4, h5 {
-  font-family: 'Source Sans Pro';
+  font-family: $su-font-sans;
 }
```

Comment:
Does this still need to be specified for the headings? It looks like the headings are getting this font-family inherited from the html already.

Reply:
All done.